src/commands/index.js

```diff
@@ -1,6 +1,5 @@
 "use strict";
 
-var previews = require("../previews");
```

Comment:
this is still needed for the other branch, please bring back. otherwise when we sync with master, it'll remove it from the other branch.